projects/tifi-bank/index.js

```diff
@@ -1,9 +1,35 @@
 const { getUniTVL } = require('../helper/unknownTokens')
-const { staking } = require('../helper/staking')
+const { sumTokensExport } = require("../helper/unknownTokens");
+const { stakings } = require('../helper/staking')
+
+const stakingContracts = [
+  // Staking in the RESERVIOR
+  "0x0AEfF3d761F6706295f3828C87ccE29c9418a93B",
+  // Loan
+  "0x8A6F7834A9d60090668F5db33FEC353a7Fb4704B",
+  // Lock to Earn
+  "0xA015263066da13e94526a8b897eDB0E3cd55B19A"
+];
+
+const lpContract = [
+  //PinkLock TIFI Bank 
+  "0x407993575c91ce7643a4d4ccacc9a98c36ee1bbe",
+  //PinkLock Cake
+  "0x7ee058420e5937496f5a2096f04caa7721cf70cc"
+]
+
+const lpTokens = [
+  // TiFi bank LP token
+  "0x707B6F02fFC0C7fD9fe3a4F392Aef47218021337",
+  // Cake LP token
+  "0xB62BB233Af2F83028Be19626256A9894B68AAe5E"
+]
+
 
 module.exports = {
   bsc: {
     tvl: getUniTVL({ chain: 'bsc', factory: '0xb3456550c17128ca7ebbcc47d4be6cae29d43853', }),
-    staking: staking('0x0AEfF3d761F6706295f3828C87ccE29c9418a93B', '0x17E65E6b9B166Fb8e7c59432F0db126711246BC0', 'bsc')
+    staking: stakings(stakingContracts, '0x17E65E6b9B166Fb8e7c59432F0db126711246BC0', 'bsc'),
+    pool2: sumTokensExport({ tokens: lpTokens, owners: lpContract, useDefaultCoreAssets: true }),
```

Comment:
sorry, I dont think tokens locked in pinksale can be considered your pool2, it needs to be LPs deposited by users in contracts controlled by you.

Reply:
@g1nt0ki Even those locks are all created by us?

Reply:
We are using the pink lock to add liquidity for the pool we use in our Dapp and show our users the numbers and dates we locked.

Reply:
And here are our lock records on the pink sale, you can see the token numbers are exactly the same means the locks is all created by us.

https://www.pinksale.finance/pinklock/detail/0x707B6F02fFC0C7fD9fe3a4F392Aef47218021337?chain=BSC
https://www.pinksale.finance/pinklock/detail/0xB62BB233Af2F83028Be19626256A9894B68AAe5E?chain=BSC

Reply:
> @g1nt0ki Even those locks are all created by us?

*sorry, no, we have not done this for other projects. LP tokens locked by team shows commitment to the project but cant be considered pool2 https://docs.llama.fi/list-your-project/what-to-include-as-tvl

Reply:
@g1nt0ki got it, I just commit the code to remove the pool2, please help us merge the PR, thank you!